### PR TITLE
Update readme

### LINKED ISSUE #3: Fill out readme with more details
@aslo I think we have plenty of material to copy from in your design doc. :)

### PR DESCRIPTION
**Pull request by aslo**
fixes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,17 +3,42 @@
 [![Build Status](https://travis-ci.org/GoogleCloudPlatform/runtime-builder-java.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/runtime-builder-java)
 
 A [Google Cloud Container Builder](https://cloud.google.com/container-builder/docs/) pipeline for 
-packaging Java applications into supported Google Cloud Runtime containers. It consists of several
-docker containers which are run using the [cloudbuild.yaml](cloudbuild.yaml) configuration file.
+packaging Java applications into supported Google Cloud Runtime containers. It consists of a series
+of docker containers and a [cloudbuild.yaml](cloudbuild.yaml) configuration file.
```
Comment (meltsufin) [inline README.md]: "...used as build steps..."

File: README.md [hunk 2]
```diff
@@ -3,17 +3,42 @@
...
 
-## Building
-To build all docker images:
+## Running via Google Cloud Container Builder (recommended)
+To run via Google Cloud Container Builder, first install the 
+[Google Cloud SDK](https://cloud.google.com/sdk/). Then, initiate a Cloud Container Build:
```
Comment (meltsufin) [inline README.md]: Can you link to the cloudbuild.yaml in this paragraph?

File: README.md [hunk 3]
```diff
@@ -3,17 +3,42 @@
...
 ```bash
-mvn package
+gcloud container builds submit /path/to/my/java/app --config cloudbuild.yaml
```
Comment (meltsufin) [inline README.md]: Say something like, "After this command is run, the built application container will appear in GCR..."

File: README.md [hunk 4]
```diff
@@ -3,17 +3,42 @@
...
 ```
 
-## Runnning Locally
-TODO
+## Configuration
+An [app.yaml](https://cloud.google.com/appengine/docs/flexible/java/configuring-your-app-with-app-yaml) 
+file must be included in the sources passed to the Java Runtime Builder. The `runtime_config`
+section of this file tells the builder how to build and package your source. In most cases, 
+`runtime_config` can be omitted.
+
```
Comment (meltsufin) [inline README.md]: Show a sample app.yaml

File: README.md [hunk 5]
```diff
@@ -3,17 +3,42 @@
...
+| Option Name | Type | Default | Description |
+|----------|------|---------|-------------|
+| artifact | string |  Discovered based on the content of your build output | The path where the builder should expect to find the artifact to package in the resulting docker container. This setting will be required if your build produces more than one artifact. 
+| build_script | string | `mvn -B -DskipTests clean package` if a maven project is detected, or `gradle build` if a gradle project is detected | The build command that is executed to build your source |
+
+## Building and Running locally
+The build pipeline can be built using maven:
```
Comment (meltsufin) [inline README.md]: Does it make sense to provide something like this also: https://github.com/GoogleCloudPlatform/openjdk-runtime/pull/63?

Comment (aslo) [inline README.md]: Might be worth looking into. I'll add an issue to track that separately.

File: README.md [hunk 6]
```diff
@@ -3,17 +3,58 @@
...
 
-## Building
-To build all docker images:
+## Running via Google Cloud Container Builder (recommended)
+To run via Google Cloud Container Builder, first install the 
+[Google Cloud SDK](https://cloud.google.com/sdk/). Then, initiate a Cloud Container Build using the 
+provided [cloudbuild.yaml](cloudbuild.yaml) file:
 ```bash
-mvn package
+gcloud container builds submit /path/to/my/java/app --config cloudbuild.yaml
```
Comment (meltsufin) [inline README.md]: Tried this and getting an error:
```
ERROR: (gcloud.container.builds.submit) INVALID_ARGUMENT: invalid build: key in the template "_OUTPUT_IMAGE" is not matched in the substitution data
```

File: README.md [hunk 7]
```diff
@@ -3,17 +3,74 @@
...
+## Running via Google Cloud Container Builder (recommended)
+To run via Google Cloud Container Builder, first install the
+[Google Cloud SDK](https://cloud.google.com/sdk/). Then, initiate a Cloud Container Build using the 
+provided [cloudbuild.yaml](cloudbuild.yaml) file:
+```bash
+# first, build locally using maven
+mvn clean install
+
+# then, push built images to GCR
+docker tag java-runtime-builder gcr.io/my-project-id/java-runtime-builder
```
Comment (meltsufin) [inline README.md]: Maybe define project as a variable?
```bash
GCP_PROJECT_ID=my-project-id
docker tag java-runtime-builder gcr.io/$GCP_PROJECT_ID/java-runtime-builder
```

**Pull request merged**